modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -269,6 +382,11 @@ export default {
         if (this.docId) {
           route = `${this.moduleAction}/${this.docId}`;
           requestMethod = apos.http.put;
+          // Make sure we fail if someone else took the advisory lock
+          body._advisoryLock = {
```

Comment:
It seems that I should be failing in my disaster scenario I laid out previously. If I took control, the initial editor should not be able to save. Or should get an error when they do.